assets/src/edit-story/elements/text/index.js

```diff
@@ -30,6 +31,7 @@ export { default as updateForResizeEvent } from './updateForResizeEvent';
 
 export const defaultAttributes = {
   ...SHARED_DEFAULT_ATTRIBUTES,
+  backgroundTextMode: BACKGROUND_TEXT_MODE.FILL,
```

Comment:
Also, it sounds like `NONE` should be default?

Reply:
Fixed in f1e5c558.